src/components/ImageDisplay/GroupImagesButton.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <div
+      v-for="(imageList, SMARTSTK) in grouped_images"
+      :key="SMARTSTK"
+    >
+      <!-- Show the last image by default -->
+      <!-- <div v-if="!showAllImages[SMARTSTK]"> -->
```

Comment:
Please correct if wrong - but AFAIK we're not using this component for this small feature, so can we remove it from this PR?

Reply:
we're not using it but I get this error when deleting key "Elements in iteration expect to have 'v-bind:key' directives."